scripts/testnet/variables.sh

```diff
@@ -122,7 +122,7 @@ export PORT_ORIGIN_VALIDATOR_REST="9500"
 export USETMUX=1
 
 # Log level for the logger in the Node.
-export LOGLEVEL="*:INFO"
+export LOGLEVEL="*:DEBUG"
```

Comment:
Revert to `INFO`?

Reply:
We always need it on debug for this feat branch, instead of always modifying it locally, will set it like this.